src/test/ui/rust-2021/future-prelude-collision-shadow.rs

```diff
@@ -0,0 +1,33 @@
+// run-rustfix
```

Comment:
```suggestion
```

The test is just meant to check that we get a compilation error -- it's not imp't to test what happens after rustfix runs. The point is that this code couldn't exist in the first place.